packages/core/src/components/forms/textArea.tsx

```diff
@@ -113,12 +126,13 @@ export class TextArea extends AbstractPureComponent<TextAreaProps, TextAreaState
 
         // add explicit height style while preserving user-supplied styles if they exist
         let { style = {} } = htmlProps;
-        if (growVertically && this.state.height != null) {
+        if (autoResize && this.state.height != null) {
             // this style object becomes non-extensible when mounted (at least in the enzyme renderer),
             // so we make a new one to add a property
             style = {
                 ...style,
                 height: `${this.state.height}px`,
+                resize: "none",
```

Comment:
one more thing -- I think this should allow horizontal resizing, because we're only controlling the vertical dimension with `autoResize`. Also this style should be set with a CSS class so that users can more easily override it if necessary.

Reply:
fixed